### PR TITLE
Add in a README and Contributing

### PR DESCRIPTION
**Pull request by paultag**

### CODE REVIEW

**Comment by paultag** [comment]
@clinty Thoughts? How's my example Haskell? (First time using the http request stuff)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    putStrLn $ case gpl of
+        Right license -> show $ (olName license)
+        Left  error   -> ("Error during license request!" ++ (show error))
+```
+
+Installing
+----------
+
+```
+cabal install opensource
```
Comment (paultag) [inline README.md:25]: Is this right?

**Comment by clinty** [comment]
Totally reasonable, but you're paren-happy.

**Comment by clinty** [comment]
(try running hlint on it)

**Comment by paultag** [comment]
@clinty neat, thanks! Fixed!

**Pull request merged**